projects/igniteui-angular/src/lib/tabs/tabs.component.ts

```diff
@@ -279,7 +279,18 @@ export class IgxTabsComponent implements IgxTabsBase, AfterViewInit, OnDestroy {
         }
     }
 
-    private resetSelectionOnCollectionChanged() {
+    private setSelectedGroup(): void {
+        requestAnimationFrame(() => {
```

Comment:
Is there a reason why we need to always set selection in animation frame? I saw a couple more `setTimeout` methods in this file. Do we really need those?

Reply:
Yes, we need requestAnimationFrame here in order to have all content rendered before selecting a group.